code/modules/mob/living/carbon/human/life/handle_breath.dm

```diff
@@ -114,8 +114,13 @@
 /mob/living/carbon/human/proc/get_breath_from_internal(volume_needed)
 	if(internal)
 		if(!contents.Find(internal))
-			internal = null
-		if(!wear_mask || !(wear_mask.clothing_flags & MASKINTERNALS))
+			if(wear_suit && isrig(wear_suit))
```

Comment:
I know this is in the body of the PR, but I personally don't like an helmet acting as a breath mask. 
This comment has no other value than to mention that.